package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-res",
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
No need to do this, we'll use `npm version` on our end which bumps this and creates a tag automatically.

Reply:
Resolved.